blog/2026-04-20-legacy-device-tracker-deprecation.md

```
@@ -0,0 +1,107 @@
+
+As of today (April 2026) most widely used core device tracker integrations have already been migrated.
+
+Note that the most popular integration which has not yet been migrated, xiaomi_miio, has a wide mix of functionality including other things than device tracker.
+
+The proposal to deprecate the legacye device tracker API was approved in [architecture proposal 1375](https://github.com/home-assistant/architecture/discussions/1375).
+
+#### List of core integrations, sorted by reported use
+
+The list was generated from https://analytics.home-assistant.io/ in March 2026
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)